common/util/autoware_point_types/test/test_point_types.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware_point_types/types.hpp"
+
+#include <gtest/gtest.h>
+
+TEST(PointXYZI, output)
```

Comment:
The usage of the TEST macro is `TEST(TestSuiteName, TestName)`.

Please see the manual.
http://google.github.io/googletest/primer.html#simple-tests

Reply:
I fixed it in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/784/commits/d501706f57795e1eea6161332101677de0a618c8